### PR TITLE
Update parameters for create_installer_linux

### PR DESCRIPTION
**Pull request by aahlenst**
This should fix the recent build failures with the linux installers. The Jenkins job `create_installer_linux` expects a parameter `RELEASE_TYPE`. The value should be `Release` if it's a release and `Nightly` otherwise.

It's my first change to the build pipeline, so please double check.

### CODE REVIEW

File: pipelines/build/common/openjdk_build_pipeline.groovy [hunk 1]
```diff
@@ -231,15 +231,20 @@ class Build {
...
         def installerJob = context.build job: "build-scripts/release/create_installer_linux",
                 propagate: true,
                 parameters: [
                         context.string(name: 'UPSTREAM_JOB_NUMBER', value: "${env.BUILD_NUMBER}"),
                         context.string(name: 'UPSTREAM_JOB_NAME', value: "${env.JOB_NAME}"),
                         context.string(name: 'FILTER', value: "${filter}"),
+                        context.string(name: 'RELEASE_TYPE', value: "${releaseType}"),
                         context.string(name: 'VERSION', value: "${versionData.version}"),
                         context.string(name: 'MAJOR_VERSION', value: "${versionData.major}"),
-                        context.string(name: 'JVM', value: "${buildConfig.VARIANT}"),
```
Comment (karianna) [inline pipelines/build/common/openjdk_build_pipeline.groovy]: Why is this removed?

Comment (aahlenst) [reply pipelines/build/common/openjdk_build_pipeline.groovy]: The type of JVM (Hotspot, OpenJ9 etc.) is now inferred from the filename of the artifacts copied into the workspace.
